### DISCUSSION TITLE
New Dynamic Tag to display the number of available posts and product

### DISCUSSION THREAD

**Discussion by Joliamendi** [elementor/elementor#21386] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi

It would be great if you add a new dynamic tag to display the number of posts. This feature will be useful for both posts and products. Of course, it will be more useful for products.

![Screenshot (1018)](https://user-images.githubusercontent.com/103376243/219697724-9389aa29-05b9-4daf-b827-bf3c58bcaefd.png)

Compatibility of this feature with loop builder would be great.

### Describe the solution you'd like

new dynamic tag to display the number of posts / product.

**Comment by 79ho3ein** [comment]
Great idea👌

**Comment by ayoubkhan558** [comment]
Dynamic tag can be created to get the total number of **posts(pages, posts, products) and taxonomies(categories, tags)** etc
